pkg/products/rhssouser/reconciler.go

```diff
@@ -186,7 +187,12 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, err
 	}
 
-	phase, err = r.CreateKeycloakRoute(ctx, serverClient, r.Config, r.Config.RHSSOCommon)
+	// Setting a name for keycloak-edge to "sso" for managed-api install type
```

Comment:
Comment doesn't match code (`"sso"` part :) )
Also, describing _why_ that is being set might be useful.

Reply:
Ah, good spot! Thanks a lot @matskiv